ivadomed/inference.py

```diff
@@ -100,6 +125,52 @@ def pred_to_nib(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False,
 
     return nib_pred
 
+def process_transformations(context, fname_roi, fname_prior, metadata, slice_axis, fname_images):
+    # If ROI is not provided then force center cropping
+    if fname_roi is None and 'ROICrop' in context["transformation"].keys():
+        print(
+            "\n WARNING: fname_roi has not been specified, then a cropping around the center of the image is "
+            "performed instead of a cropping around a Region of Interest.")
+
+        context["transformation"] = dict((key, value) if key != 'ROICrop'
+                                            else ('CenterCrop', value)
+                                            for (key, value) in context["transformation"].items())
+
+    if 'object_detection_params' in context and \
+            context['object_detection_params']['object_detection_path'] is not None:
+        imed_obj_detect.bounding_box_prior(fname_prior, metadata, slice_axis,
+                                            context['object_detection_params']['safety_factor'])
+        metadata = [metadata] * len(fname_images)
+    
+    return metadata
+
+def set_option(options, postpro, context, key):
```

Comment:
Thank you for generalizing this to make things easier in the future. Ditto above.